week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -15,52 +15,55 @@ and have different levels of positivity or negativity.
 Below are the possible answers:
 
 ## Very positive
-It is certain.
-It is decidedly so.
-Without a doubt.
-Yes - definitely.
-You may rely on it.
-
-## Positive
-As I see it, yes.
-Most likely.
-Outlook good.
-Yes.
-Signs point to yes.
-
-## Negative
-Reply hazy, try again.
-Ask again later.
-Better not tell you now.
-Cannot predict now.
-Concentrate and ask again.
-
-## Very negative
-Don't count on it.
-My reply is no.
-My sources say no.
-Outlook not so good.
-Very doubtful.
-*/
-
+**/
+let arr = [
+  "It is certain",
+  "It is decidedly so",
+  "Without a doubt",
+  "Yes - definitely",
+  "You may rely on it",
+
+  "As I see it, yes",
+  "Most likely",
+  "Outlook good",
+  "Yes",
+  "Signs point to yes",
+
+  "Reply hazy, try again",
+  "Ask again later",
+  "Better not tell you now",
+  "Cannot predict now",
+  "Concentrate and ask again",
+
+  "Don't count on it",
+  "My reply is no",
+  "My sources say no",
+  "Outlook not so good",
+  "Very doubtful",
+];
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  console.log(answer);
+  return answer;
+}
 
 // The answer should come from shaking the ball
-let answer;
+let answer = "The ball has shaken!";
 
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer(answer) {
+  return "very positive";
+}
```

Comment:
We can take about this at the end of the week. It is meant for you to check the result you got from the previous function and compare it against the array

Reply:
You're much closer.